src/content/en/resources/contributors.md

```diff
@@ -0,0 +1,1751 @@
+project_path: /web/_project.yaml
```

Comment:
this file should not be included in the PR - it's autogenerated and has been moved to `/resources/contributors/index.md`